web/src/components/tasks/cigarette-license/CigaretteLicenseStepOne.tsx

```diff
@@ -0,0 +1,42 @@
+import { Content } from "@/components/Content";
```

Comment:
[dust] I'd rename this component `GeneralInfo` or `StepOne`. My preference would be `GeneralInfo.